scripts/triggerShortcut.ts

```diff
@@ -71,6 +99,10 @@ void (async () => {
   // or at least a list of the expected fields
   // This just throws a Zod error if the input is not valid
   const triggerInput = z.object(shortcut.triggerInputShape).parse(argv)
+  // For deposit/withdraw shortcut, tokens field won't parse correctly. Instead, pass it in as
+  // a string and change the above to: {...shortcut.triggerInputShape, tokens: z.string()}
+  // Then below under ...triggerInput, add tokens: JSON.parse(triggerInput.tokens)
+  // Same happens for swap-deposit shortcut, but with swapFromToken field
```

Comment:
would be nice to get this working automatically, wonder if we could just pass all of triggerInputShape as a json string and then do a JSON.parse and validate against the schema. @jeanregisser ideas?

Reply:
updated in [c56a987](https://github.com/valora-inc/hooks/pull/593/commits/c56a987fc765c8a9dbb817ef9f7cb727fa5665f8), example using it in the description

Reply:
Good idea @satish-ravi 👍